src/app/_components/ForgotPassword/ForgotPassword.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Box } from "@mui/material";
+import ForgotPassword from "./ForgotPassword";
+
+const meta = {
+  title: "Authentication/ForgotPassword",
+  component: ForgotPassword,
+  parameters: {
+    layout: "centered",
+  },
+  tags: ["autodocs"],
+  argTypes: {
+    email: { control: "text" },
```

Comment:
Your build failure is saying this `email` field doesn't exist in your component props type. 
![image](https://github.com/Re-boot-Coding-Bootcamp/TwistaGram/assets/24844451/47e5cba8-cd02-4c38-a15c-394eb4e8a095)